site/docs/voice/bxml/startTranscription.mdx

```diff
@@ -0,0 +1,242 @@
+If the `transcriptionEventUrl` attribute is specified, then the [StartTranscription Available](5) event will be sent once the transcription has ended and the transcription is available at `/accounts/{accountId}/calls/{callId}/transcriptions/{transcriptionId}`. BXML returned in response to this callback will be ignored.
+
+:::note
+While multiple real-time transcriptions for the same call are allowed, each real-time transcription MUST have a unique name. Attempting to start a real-time transcription on the same call with the name of an already existing real-time transcription will result in a [Real-Time Transcription Rejected][3] event.
+:::
+
+## Webhooks Received
+
+| Webhooks                   | Can reply with more BXML |
+|:---------------------------|:-------------------------|
+| [Real-Time Transcription Started][2]  | No                       |
+| [Real-Time Transcription Rejected][3] | No                       |
+| [Real-Time Transcription Stopped][4]  | No                       |
+
+## Nested Tags
+
+You may specify up to 12 `<CustomParam/>` elements nested within a `<StartTranscription>` tag. These elements define optional user specified parameters that will be sent to the destination URL when the real-time transcription is first started.
+
+### CustomParam Attributes
+
+| Attribute | Description                                                    |
+|:----------|:---------------------------------------------------------------|
+| name      | (required) The name of this parameter, up to 256 characters.   |
+| value     | (required) The value of this parameter, up to 2048 characters. |
+
+## Websocket Packet Format
+
+If a `destination` is specified, it will be sent JSON messages for the duration of the real-time transcription. There will be an initial `start` message when the connection is first established. This will be followed by zero or more `transcription` messages containing transcription updates for the tracks being transcribed. Finally, when a real-time transcription is stopped, a `stop` message will be sent.
+
+### Start and Stop Message Parameters
+
+| Parameter    | Description |
+|:-------------|:------------|
+| eventType    | What type of message this is, one of `start`, or `stop` |
+| metadata     | Details about the real-time transcription this message is for. See further details below. |
+| customParams | (optional) (`start` message only) If any `<CustomParam/>` elements were specified in the `<StartTranscription>` request, they will be copied here as a map of `name : value` pairs |
+
+#### Metadata Parameters
+
+| Parameter                     | Description |
+|:------------------------------|:------------|
+| accountId                     | The user account associated with the call |
+| callId                        | The call id associated with the real-time transcription |
+| realtimetranscriptionId       | The unique id of the real-time transcription |
```

Comment:
```suggestion
| realTimeTranscriptionId       | The unique id of the real-time transcription |
```
Also, is this not just `transcriptionId` so that customers don't mix them with batch transcriptions?

Reply:
Yes, but I'm fine with use `transcriptionId` if we think that's a good idea.